pkg/registry/apis/alerting/notifications/timeinterval/conversions.go

```diff
@@ -10,6 +12,12 @@ import (
 	"github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
 )
 
+func getIntervalUID(t definitions.MuteTimeInterval) string {
```

Comment:
Q: Is the required spec here RFC 1035 or RFC 1123, if it's RFC 1035 then this isn't a valid UID as it can start with a number?

Reply:
From what I understand in [this article](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/), K8s does not enforce RFC 1035 but leaves it as  opt-in. I am not aware of any restrictions in app-sdk, and looking at other APIs that use just UID (folders, playlists), I think we could follow less restrictive RFC 1123.